zerver/lib/message.py

```diff
@@ -1507,16 +1506,14 @@ def stream_wildcard_mention_allowed(sender: UserProfile, stream: Stream, realm:
     return can_mention_many_users(sender)
 
 
-def check_user_group_mention_allowed(sender: UserProfile, user_group_ids: list[int]) -> None:
```

Comment:
I safely removed `sender` parameters, as `mention_data` is already tied to single message for a single user.